discovery-infra/test_infra/consts/consts.py

```diff
@@ -40,6 +40,8 @@
 DEFAULT_NAMESPACE = 'assisted-installer'
 DEFAULT_TEST_INFRA_DOMAIN = f".{CLUSTER_PREFIX}-{DEFAULT_NAMESPACE}.{DEFAULT_BASE_DNS_DOMAIN}"
 TEST_TARGET_INTERFACE = "vnet3"
+RANGE_8BIT = 256
```

Comment:
`2**8`

Reply:
2**8 + 1, I want it to effect the second bit

Reply:
I added the +1 to the calculation itself

Reply:
But 256 is exactly 2**8, not 2**8+1

2**8+1 is 257

Reply:
oh ops, you are right, fixing